docs/source/tutorials/artifact.rst

```diff
@@ -0,0 +1,258 @@
+========
+Artifact
+========
+
+A data artifact is a bundle of input data associated with a particular
+model. It is typically stored as an ``hdf`` file on disk with very particular
+formatting. This file is then used by the :mod:`vivarium` simulations to fill
+in all the relevant parameter data.
+
+It is frequently useful to be able to view or modify this data outside the
+simulation.  The :class:`vivarium.Artifact` provides a high level interface to
```

Comment:
elsewhere you seem to use :class:`~vivarium.Artifact` but here you're not using the tilda - does that matter?

Reply:
The tilda removes leading import paths.  I left the full path in since it's the first mention.